access/api.go

```diff
@@ -32,6 +32,7 @@ type API interface {
 	GetAccount(ctx context.Context, address flow.Address) (*flow.Account, error)
 	GetAccountAtLatestBlock(ctx context.Context, address flow.Address) (*flow.Account, error)
 	GetAccountAtBlockHeight(ctx context.Context, address flow.Address, height uint64) (*flow.Account, error)
+	GetAccountAtBlockID(ctx context.Context, address flow.Address, blockID flow.Identifier) (*flow.Account, error)
```

Comment:
Just syntax sugar here but rather than clubbing it here with the existing interface, we could define a new interface -  `UnstakedNetworkAPI` interface which defines this method `GetAccountAtBlockID`. It will just help make it clear that there are two different sets of interface and an access node may support one or both.